x-pack/test/saved_object_api_integration/security_and_spaces/apis/import.ts

```diff
@@ -20,27 +20,78 @@ const {
+  // an error; otherwise, we expect to receive a success result
+  const cases = Object.entries(CASES).filter(([key]) => key !== 'HIDDEN');
+  const importable = cases.map(([, val]) => ({ ...val, successParam: 'trueCopy' }));
+  const nonImportable = [{ ...CASES.HIDDEN, ...fail400() }];
+  const all = [...importable, ...nonImportable];
+  return { importable, nonImportable, all };
+};
+
+const createTestCases = (overwrite: boolean, spaceId: string) => {
   // for each permitted (non-403) outcome, if failure !== undefined then we expect
   // to receive an error; otherwise, we expect to receive a success result
-  const importableTypes = [
-    { ...CASES.SINGLE_NAMESPACE_DEFAULT_SPACE, ...fail409(spaceId === DEFAULT_SPACE_ID) },
-    { ...CASES.SINGLE_NAMESPACE_SPACE_1, ...fail409(spaceId === SPACE_1_ID) },
-    { ...CASES.SINGLE_NAMESPACE_SPACE_2, ...fail409(spaceId === SPACE_2_ID) },
-    { ...CASES.NAMESPACE_AGNOSTIC, ...fail409() },
+  const group1Importable = [
+    // when overwrite=true, all of the objects in this group are created successfully, so we can check the created object attributes
+    {
+      ...CASES.SINGLE_NAMESPACE_DEFAULT_SPACE,
+      ...fail409(!overwrite && spaceId === DEFAULT_SPACE_ID),
+    },
+    { ...CASES.SINGLE_NAMESPACE_SPACE_1, ...fail409(!overwrite && spaceId === SPACE_1_ID) },
+    { ...CASES.SINGLE_NAMESPACE_SPACE_2, ...fail409(!overwrite && spaceId === SPACE_2_ID) },
+    { ...CASES.NAMESPACE_AGNOSTIC, ...fail409(!overwrite) },
     CASES.NEW_SINGLE_NAMESPACE_OBJ,
     CASES.NEW_NAMESPACE_AGNOSTIC_OBJ,
   ];
-  const nonImportableTypes = [
-    { ...CASES.MULTI_NAMESPACE_DEFAULT_AND_SPACE_1, ...fail400() },
-    { ...CASES.MULTI_NAMESPACE_ONLY_SPACE_1, ...fail400() },
-    { ...CASES.MULTI_NAMESPACE_ONLY_SPACE_2, ...fail400() },
-    { ...CASES.HIDDEN, ...fail400() },
-    { ...CASES.NEW_MULTI_NAMESPACE_OBJ, ...fail400() },
+  const group1NonImportable = [{ ...CASES.HIDDEN, ...fail400() }];
+  const group1All = group1Importable.concat(group1NonImportable);
+  const group2 = [
+    // when overwrite=true, all of the objects in this group are created successfully, so we can check the created object attributes
+    CASES.NEW_MULTI_NAMESPACE_OBJ,
+    {
+      ...CASES.MULTI_NAMESPACE_DEFAULT_AND_SPACE_1,
+      ...fail409(!overwrite && (spaceId === DEFAULT_SPACE_ID || spaceId === SPACE_1_ID)),
+      ...destinationId(spaceId !== DEFAULT_SPACE_ID && spaceId !== SPACE_1_ID),
+    },
+    {
+      ...CASES.MULTI_NAMESPACE_ONLY_SPACE_1,
+      ...fail409(!overwrite && spaceId === SPACE_1_ID),
+      ...destinationId(spaceId !== SPACE_1_ID),
+    },
+    {
+      ...CASES.MULTI_NAMESPACE_ONLY_SPACE_2,
+      ...fail409(!overwrite && spaceId === SPACE_2_ID),
+      ...destinationId(spaceId !== SPACE_2_ID),
+    },
+    { ...CASES.CONFLICT_1A_OBJ, ...newOrigin() }, // "ambiguous source" conflict which results in a new destination ID and empty origin ID
```

Comment:
observation: These comments are appreciated, as they help my understand the various test scenarios. To say it another way, I'd be lost without them...I know you've put a ton of work into these test suites recently too, but I do worry about the maintainability of these scenarios. It takes so long to grok these before you can start making changes.

I also know that this problem predates you, so please don't think I'm pointing fingers :)

As a team, we should take some time to reevaluate these test suites, and see if there's a way we can get the same (or similar) coverage, with better readability